### PR TITLE
Add FILWallet information

### PR DESCRIPTION
**Pull request by filwallet**
FILWallet is Crypto wallet focusing on FIL. The application has been online for almost a year now. FILWallet has integrated the following features：
    1. Supports the FIL full account system (f0, f1, f2, f3) to provide the most professional.
    2. Support multi-signature
    3. Support node management, can change Owner to multi-signature, node withdrawal, observe node dynamics.
    4. Support the function of re-signing after the failure of multi-signature signature.
    5. Support message push, unified management of push content.
    6. Support international languages (English, Traditional Chinese, Korean, Japanese).
    7. Support multi-signature red envelope function, red envelopes are distributed through multi-signature addresses.

### CODE REVIEW

**Comment by filwallet** [comment]
FILWallet currently has 3K+ user groups, and the daily transfer FIL is nearly 10W+, And received unanimous praise from ecological users.
FILWALLET attaches great importance to the security of wallets and has completed the security audit of CERTIK and artemisclouds

**Comment by filwallet** [comment]
Some screenshots of FILWallet
image
image
image
image
image
image
image
image

**Comment by johnnymatthews** [comment]
Sounds good. Please link to the audits performed against FILWallet.

**Comment by filwallet** [comment]
> Sounds good. Please link to the audits performed against FILWallet.

https://www.certik.com/projects/filwallet

**Comment by johnnymatthews** [comment]
We're concerned about CertiK's audit, mainly because they've failed to detect rug-pulls in past audits. We're still investigating this wallet.

**Comment by filwallet** [comment]
@johnnymatthews OK, our team will cooperate fully and hope to get official recognition as soon as possible to help the FIL ecosystem, thank you.

**Comment by filwallet** [comment]
@johnnymatthews How is the progress, do you need the cooperation of the filwallet team?

**Comment by johnnymatthews** [comment]
Thanks for your patience. We will be adding this wallet to Filecoin Docs following the completion of https://github.com/filecoin-project/filecoin-docs/issues/1195 (this will be completed by someone internal at Filecoin, likely complete at some point this week).

**Comment by filwallet** [comment]
@johnnymatthews OK. Thanks.

File: docs/about-filecoin/managing-assets.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Be incredibly cautious when using unaudited wallets. Wallets that _have_ gone th
 | [MathWallet](https://mathwallet.org/en-us/)    | A multi-currency wallet.                                                                                                             | Unknown |
 | [Trust wallet](https://trustwallet.com/)                                   | A multi-currency wallet. This is the official wallet of Binance.                                                                     | Unknown |
 | [FoxWallet](https://foxwallet.com/)            | A multi-currency mobile wallet by Filfox                                                                                             | Yes     |
+| [FILWallet](https://filwallet.co)            | Crypto wallet focusing on FIL | Yes     |
 
 ## Exchanges
```
Comment (DSBUGAY2) [inline docs/about-filecoin/managing-assets.md:28]: f1ihg2wkwz4pxngi723yvncyyeqvqju3r6cu5iray

**Review by DSBUGAY2 — APPROVED** [review]

**Comment by DSBUGAY2** [comment]
f1ihg2wkwz4pxngi723yvncyyeqvqju3r6cu5iray

**Comment by johnnymatthews** [comment]
FilWallet information is now displayed. 

image

Closing.

**Pull request closed without merging**